app/assets/stylesheets/style.scss

```diff
@@ -1,3 +1,31 @@
+body {
+  background-image: url("http://i.imgur.com/y6o43z3.jpg");
```

Comment:
Thank you Richard. When I was implementing last night I was trying to link images how I would in regular html with no success, and it got late, so I got lazy and used imgur. Christina, totally agree! At this point I was just trying to throw together a visual identity for the group to look at and give me input on. My plan is to change them to html/css, but I'm still new to Foundation, so if you'd like to go over that with me I would greatly appreciate it!